cmd/wallet-web/src/mixins/gnap/gnap.js

```diff
@@ -36,7 +24,7 @@ export async function gnapRequestAccess(signer) {
       start: ['redirect'],
```

Comment:
need followup : `store.getters()` is vuex specific, these vue specific code shouldn't be part of mixin package which is pure javascript.

Reply:
removed from mixin package